Automattic/wp-calypso
# /plugins: Integrate the LOHP footer section

#### Proposed Changes

* /plugins: Integrate the LOHP footer section
* 1001-gh-Automattic/martech

#### Testing Instructions

* Log out
* Navigate to /plugins
* Ensure that you see the sitewide Footer Nav bars.
* The page should be unaffected if logged in


#### Pre-merge Checklist






Related to #
